pre-commit/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache git npm bash build-base && \
+    npm config set unsafe-perm true && \
+    pip install pre-commit
+
+CMD ["pre-commit"]
+
+LABEL name=pre-commit version=1.0 \
```

Comment:
This is not true, the current version, which is the one that gets installed it's `1.18.3`

Reply:
Not sure about the value of that label tbh